sdk/src/generated-client.ts

```diff
@@ -0,0 +1,2976 @@
+/**
```

Comment:
This generated code shouldn't be a part of GitHub repository, it should be `.gitignore`d.

Instead, you may add it to `"build"` script in NPM.